### ISSUE TITLE
v2.19 add double script-src in Content-Security-Policy Header, breaking /sheet

### ISSUE THREAD

**Issue by MacLemon** [xwiki-labs/cryptpad#348]
# Summary:
Sheets are broken in v2.19 due to a double script-src CSP.

# Steps to Reproduce:
Use 2.19 open `https://example.com/sheet/`. Sheet starts to load, but stops somewhere before loading the “getting started” text. 

# Expected Results:
Describe what you expected to happen when you executed the steps above.

Actual Results:
Sheets should load. Browser console complains about an incorrect `frame-ancestors`
 policy in CSP, which is due to the fact that a surplus `script-src` get's added after the frame-ancestors without the separating `;`.

# Regression:
This is new in v2.19, it worked ok in 2.18.

# Notes:
One can workaround the `frame-ancestors` header by adding a `;` in `config.js` in line 47 so it reads `"frame-ancestors * ;"`. This will remove the syntax error in the CSP, but will earth the double `script-src` in the CSP which will then be ignored by the browser and it falls back to the `default-src` for javascript which is `none`.
Probably #343 is related to this.

# Version Information:
Cryptpad 2.19
Chromium 73.0.3637.0

**Comment by ineiti** [comment]
I tried `staging`, and still get this in my browser:
```
Ignoring duplicate Content-Security-Policy directive 'script-src'.

index.html:179 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https://my.real.domain/". Either the 'unsafe-inline' keyword, a hash ('sha256-my-base64-hash='), or a nonce ('nonce-...') is required to enable inline execution.
```

**Comment by ineiti** [comment]
Made a cleaner fix (IMHO) in #353 . Problem is: the `config.js` is invalid. So if somebody already started a cryptpad server, he'll have an invalid `config.js` :(
